Source/WebCore/platform/graphics/ImageBuffer.cpp

```diff
@@ -526,8 +539,10 @@ bool ImageBuffer::isInUse() const
 
 void ImageBuffer::releaseGraphicsContext()
 {
-    if (auto* backend = ensureBackendCreated())
-        return backend->releaseGraphicsContext();
+    if (!m_context)
+        return;
+    m_context = nullptr;
+    backend()->contextReleased();
```

Comment:
This is crashing because `takeBackend` can remove the backend, without clearing the context.